README.md

```diff
@@ -496,6 +496,204 @@ We expect some apps will need some features that are not currently available:
 
 We'd love to hear your feedback on these or other features. Please feel free to [open a discussion](https://github.com/gladly-team/next-firebase-auth/discussions)!
 
+## Test examples
+
+### Jest
+
+In order to test components wrapped with functions from `next-firebase-auth`, you will need to mock the `next-firebase-auth` library. This can be achieved using the [manual mocks feature of Jest](https://jestjs.io/docs/manual-mocks).
+
+To start you need to stub out the entire module in a `__mocks__` folder next to `node_modules` in your application, eg.
```

Comment:
I'm going to soften this language a bit, because there are other approaches someone could use to testing/mocking.

Reply:
Will also note that the manual mocks are helpful for defining default behavior but not strictly necessary.